apps/admin-portal/src/components/identity-providers/forms/general-details-form.tsx

```diff
@@ -73,14 +74,12 @@ interface GeneralDetailsFormPopsInterface {
  *
  * @param props GeneralDetailsFormPopsInterface.
```

Comment:
```suggestion
 * @param props GeneralDetailsFormPopsInterface.
 * @return {React.ReactElement}
```

Reply:
Fixed in cb34564693f1d730c6a791813685e95dd490c094.